src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ activeTabId, onTabSelected, tabs }) => {
+  const visibleTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': visibleTab.id === tab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() =>
+                  visibleTab.id !== tab.id && onTabSelected(tab.id)
+                }
+              >
+                {tab.title}
+              </a>
+            </li>
```

Comment:
Each element in a list should have a unique `key` prop to help React identify which items have changed, been added, or removed. This is crucial for performance optimization and avoiding potential bugs. Please add a `key` prop to the `<li>` element, using a unique identifier such as `tab.id`.